packages/static/kairos-overlay-files/files/system/oem/00_home_dir_owner_fix.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Fix home directory permissions (kairos issue #2797)"
+stages:
+  initramfs.after:
```

Comment:
I wonder if we should somehow make it an one-off fix. This will now run on every boot. With a sentinel file or something. Or maybe we should add a check and only do the `chown` if the owner is not the expected one?

Reply:
ok adapted the script to do that.

Reply:
nope. This doesn't work. The home directory itself was already owned by `kairos`. It's those files inside that were wrong (e.g. `authorized_keys` in the `.ssh` directory). I'll revert.